src/common/environment.ts

```diff
@@ -99,8 +99,16 @@ export const environment = {
   logbookClaimerPrivateKey:
     process.env.MATTERS_LOGBOOK_CLAIMER_PRIVATE_KEY || '',
   alchemyApiKey: process.env.MATTERS_ALCHEMY_API_KEY || '',
+  curationContractAddress:
+    process.env.MATTERS_CURATION_CONTRACT_ADDRESS ||
+    (isProd ? '' : '0xa219C6722008aa22828B31A13ab9Ba93bB91222c'),
```

Comment:
why hardcode?

Reply:
exclude those private envs like privateKey...

Reply:
can put them in `.env.example`?